pkg/cmd/kafka/describe/describe.go

```diff
@@ -125,6 +129,11 @@ func runDescribe(opts *Options) error {
 		}
 	}
 
+	if opts.bootstrapServer {
+		fmt.Fprintf(os.Stdout, "%v", *kafkaInstance.BootstrapServerHost)
```

Comment:
```suggestion
		fmt.Println(os.Stdout, kafkaInstance.GetBootstrapServerHost())
```

^ Untested, but this is preferred. `IOStreams` is a an abstraction of stderr and stdout which we use instead.

Reply:
It is possible for the bootstrap server URL to be nil. We should handle this with a message.

Reply:
Done, but we still have to use fmt.Fprintf function
`fmt.Fprintf(opts.IO.Out, kafkaInstance.GetBootstrapServerHost())`